### ISSUE TITLE
Fix for latest API changes

### ISSUE THREAD

**Issue by gergy008** [colinbendell/homebridge-blink-for-home#128]
With the latest API changes, I found that the plugin no longer works.

However, I have been able to get it to work but copying the latest info from the app. Using a man-in-the-middle proxy, I was able to retrieve the following info:

/* - Pulled from the app on 8/10/2022 - country was already blank
{
"client_name":"iPhone",
"password":"-snip-",
"notification_key":"-snip-",
"client_type":"ios",
"email":"-snip-",
"country":""
,"app_version":"6.17.0 (2209271715) #e459ba4a8-mod",
"unique_id":"-use your own iPhone uuid-",
"device_identifier":"iPhone14,2",
"os_version":"16.0"
}

And I changed the headers to:
        const headers = {
            'User-Agent': 'Blink/2209271715 CFNetwork/1390 Darwin/22.0.0',
            'APP-BUILD': 'IOS_2209271715',
            'LOCALE': 'en_GB',
            'x-blink-time-zone': 'Europe/London',
            'accept-language': 'en-GB',
            'Accept': '*\/*',
        };
*/ 

I used my iPhone to rip this, I would recommend using a proxy to sniff out your own device info and swap it out incase Blink is verifying this. Of course, because I had my UUID from the MITM attack I didn't need to 2fa either, and I wasn't getting an email.
Remember to change your locale, you can find values for where you live by googling http header locale for example.

**Comment by colinbendell** [comment]
Thanks. I'll look into this later today and push a new version.

**Comment by njradford** [comment]
Is this by chance related to: https://github.com/colinbendell/homebridge-blink-for-home/issues/119 ?

**Comment by gergy008** [comment]
Yes I believe it could beSent from my iPhoneOn 16 Oct 2022, at 02:43, Nick R ***@***.***> wrote:﻿
Is this by chance related to: #119 ?

**Comment by McPaddy85** [comment]
Is there any update on this issue @colinbendell ?

Tried adding blink to my HOOBS today and no luck.
